package.json

```diff
@@ -5,6 +5,8 @@
   "author": "Node.js Dev Team <https://github.com/nodejs/nodejs.dev#the-nodejsdev-team>",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.4.0",
```

Comment:
Late last year there was a big effort to [remove emotion](https://github.com/nodejs/nodejs.dev/pull/928) from this project. Perhaps we can use a regular object here instead for styling?

Reply:
Agree, looks like calendar component supports regular objects as input param